printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/README.md

```diff
@@ -0,0 +1,24 @@
+This is my Skirt Mod to add a FYSETC-Mini-12864-Panel Display, Version 2.1 to the V0
+ 
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/03.JPG?raw=true)
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/04.JPG?raw=true)
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/02.JPG?raw=true)
+![alt text](https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/01.JPG?raw=true)
+
+- The cut-out in the Back Skirt is for a 
+Schaffner FN282-2-06 DC Power Connector with filter:
+https://uk.rs-online.com/web/p/iec-filters/1704958/
+
+- The back skirt is also designed for a Raspberry Pi 4
+
+- All parts are also available as a step file if you need to modify or adapt them
+
+- Add the display chapter to your printer.cfg in klipper (<a href="https://github.com/PurchenZuPoden/VoronUsers/blob/master/printer_mods/PurchenZuPoden/V0SKIRT_W_DISPLAY/Display_Config.txt" target="_blank">Display_Config.txt</a>)
```

Comment:
Please make this a relative link

    [Display_Config.txt](Display_Config.txt)

Reply:
Code is now integrated in the readme. Fixed